### PR TITLE
AS-1708: Add function for checking if a bounding box is extremely large

### PR DESCRIPTION
**Pull request by mcamarad**
## Why

We are having issues with bounding boxes that are incorrect and they span around the world. This PR adds a function to check that they don't go beyond certain limits. Another PR in the nlp-outbreak-detection system is required to apply this to new samples

## What
- Adds function to check if bounding box stretches around the world
- Updated test for new function

### CODE REVIEW

File: tests/test_bounding_box.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from geocoder_module.geocoder import Geocoder
+
+geocoder = Geocoder()
+
+
+class TestBoundingBoxIsLarge:
```
Comment (ericanand) [inline tests/test_bounding_box.py:8]: Should we also have test cases for when `x` and `y` are both large/not large, and when 1 of each of `x` and `y` is large/not large? Is that even possible? It just occurred to me to check that is all 😁

File: geocoder_module/geocoder.py [hunk 2]
```diff
@@ -659,3 +659,23 @@ def merge_bounding_boxes(self, bounding_boxes: List[List[float]]) -> List[float]
...
+        others.
+
+        :param bounding_box:   bounding boxes, list of four
+                               float (x1,y1,x2,y2)
+        """
+        x1, y1, x2, y2 = bounding_box
+
+        # Make sure that no pair of coordinates goes under -175 and over 175 in both ends.
+        # Sometimes the coordinates come flipped so we'll check each pair.
+        if ((x1 <= -175) and (x2 >= 175)) or ((x2 <= -175) and (x1 >= 175)):
```
Comment (ericanand) [inline geocoder_module/geocoder.py:676]: So if I came in knowing absolutely nothing about this - I would be asking why `175` in particular? Could we please add a comment about why that number has been picked? 🙏

File: requirements.txt [hunk 3]
```diff
@@ -1,2 +1,4 @@
 requests==2.25.1
 pytest==6.1.1
+asyncio==3.4.3
+pytest-asyncio==0.18.3
```
Comment (ericanand) [inline requirements.txt:4]: Just checking why these have been added? Don't see us using this in this PR?

File: pytest.ini [hunk 4]
```diff
@@ -0,0 +1,2 @@
+[pytest]
+asyncio_mode = auto
```
Comment (ericanand) [inline pytest.ini:2]: Also same here - are we using this?

**Review by ericanand — COMMENTED** [review]
Some minor comments.

File: requirements.txt (hunk 3 above)
Comment (mcamarad) [reply requirements.txt:4]: Sorry, I was tinkering with asyncio and forgot to remove those. Same with the comment below

File: tests/test_bounding_box.py (hunk 1 above)
Comment (mcamarad) [reply tests/test_bounding_box.py:8]: I think I've covered all scenarios. Thanks for the great suggestion! 👍

**Pull request merged**